docs/source/guides/session_limits.rst

```diff
@@ -0,0 +1,75 @@
+How To Set Orchestrator Session Limits
+======================================
+
+What Are Orchestrator Sessions?
+******************************
+
+By default an Orchestrator node can handle a maximum of **10** incoming livestreams at a time. Whenever the number of concurrent transcoding sessions go above this, the Orchestrator returns an ``OrchestratorCapped`` error to the Broadcaster node.
+
+Of course, the transcoding capacity and the bandwidth an Orchestrator has varies for everyone - so you can set the ``-maxSessions`` cli parameter with the correct number to fully utilize their hardware and get maximum work from the network.
```

Comment:
I wouldn't characterize the value provided here as "correct" so much as a value that better reflects the orchestrator's true capacity. What about something like:

"CLI parameter with a value that reflects available transcoding capacity and bandwidth to maximize work received from broadcasters."